lib/compression.js

```diff
@@ -17,10 +18,11 @@ const PARQUET_COMPRESSION_METHODS = {
     deflate: deflate_snappy,
     inflate: inflate_snappy
   },
-  'LZO': {
-    deflate: deflate_lzo,
-    inflate: inflate_lzo
-  },
+  // Disabled. See: https://github.com/LibertyDSNP/parquetjs/issues/18
```

Comment:
I'm a little torn about leaving lzo comments for future reference. I'm lean towards removing them though. There's nothing in here that would be difficult to reimplement. Also is seems a little far fetched at this point that LZO ever comes back.

Reply:
I think it would be fine to remove the commented out code, but it's part of the parquet spec, so not fixing it at all would mean that the library would be out of spec, so I would like to at least leave in the link to the Issue.

Reply:
re: meta_data.bloom_filter_offset - this behavior isn't happening any more after I put it back.  Heisenbug? Or maybe this was a red herring.